### PR TITLE
Introduce kyverno policies

### PR DESCRIPTION
**Pull request by filariow**
Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

**Review by sadlerap — COMMENTED** [review]
Largely lgtm now that the issue with labels is fixed, but I do have a question.

File: e2e/Makefile [hunk 1]
```diff
@@ -43,7 +75,8 @@ deploy-operator:
 					--from-literal=kubesaw.namespace=$$(( [[ -n "$$toolchain_host" ]] && echo "$$toolchain_host" ) || echo "workspaces-system" ) \
 			) \
 		) && \
-		IMG=$(OPERATOR_IMG) $(MAKE) install deploy \
+		cd "$$f/e2e/config/operator/e2e" && $(KUSTOMIZE) edit set image controller=${OPERATOR_IMG} && \
+		$(KUSTOMIZE) build "$$f/e2e/config/operator/e2e" | kubectl apply -f - \
```
[the remark this answers is not in the document]
Comment (filariow) [reply e2e/Makefile:79]: because here we are using an overlay that lives at `e2e/config/operator/e2e` and changes the user workspace quota from 1 to 3

**Comment by filariow** [comment]
/do-not-merge/work-in-progress

**Comment by sadlerap** [comment]
Does it make sense to keep this PR around?  It's going to require a significant amount of work to rebase this, and I don't see us prioritizing this work anytime soon.

@filariow thoughts?

**Pull request closed without merging**